Dockerfile.win32

```diff
@@ -62,21 +62,22 @@ RUN ./autogen.sh && \
 FROM debian:latest
 
 # install wine for 32-bit architecture
-RUN apt-get update && dpkg --add-architecture i386 && apt-get update && apt-get install -y \
+RUN apt-get update && dpkg --add-architecture i386 && apt-get update && apt-get -y dist-upgrade && apt-get install -y \
     fonts-wine \
     libc6-dev-i386-x32-cross \
     mingw-w64 \
     mingw-w64-i686-dev \
     wine \
     wine32 \
     unzip \
-   && rm -rf /var/lib/apt/lists/*
+   && apt-get clean && rm -rf /var/lib/apt/lists/*
 
 ENV SRCDIR=/usr/src/liblouis
+env LOUIS_TABLEPATH=/usr/src/liblouis/share/liblouis/tables
 WORKDIR ${SRCDIR}
 COPY --from=builder ${SRCDIR}/liblouis.zip .
 RUN unzip liblouis.zip
 
-# just run any old self-contained yaml test that doesn't need any env setup
-ADD tests/yaml/letterDefTest_harness.yaml .
-RUN wine bin/lou_checkyaml.exe letterDefTest_harness.yaml
+# run spaces.yaml test to we ensure cross-compile build works right with 32 bit Windows versions
```

Comment:
@hammera I think you explained this somewhere else but I don't really understand why the tests should work when the spaces test instead of the letterDefTest_harness test. If the spaces test fails we should find out why instead of just using a different test.